1.1/geo.ttl

```diff
@@ -1,724 +1,492 @@
+@prefix geosparql-spec: <http://www.opengis.net/spec/geosparql/1.1> .
 @prefix owl: <http://www.w3.org/2002/07/owl#> .
-@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
-@prefix xml: <http://www.w3.org/XML/1998/namespace> .
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sdo: <https://schema.org/> .
 @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
 
-<http://www.opengis.net/ont/geosparql> a owl:Ontology ;
-    dc:title "GeoSPARQL Ontology" ;
-	dc:creator "Open Geospatial Consortium"^^xsd:string ;
-	dc:date "2012-04-30"^^xsd:date ;
-	dc:description "An RDF/OWL vocabulary for representing spatial information"^^xsd:string ;
-	dc:source <http://www.opengis.net/doc/IS/geosparql/1.0> , "OGC GeoSPARQL – A Geographic Query Language for RDF Data OGC 11-052r5"^^xsd:string ;
-	rdfs:seeAlso <http://www.opengis.net/def/function/ogc-geosparql/1.0> , <http://www.opengis.net/def/rule/ogc-geosparql/1.0> , <http://www.opengis.net/doc/IS/geosparql/1.0> ;
-	owl:versionInfo "OGC GeoSPARQL 1.0"^^xsd:string .
-
-# 
-# 
-# 
+geosparql: 
+	a owl:Ontology ;
+    dcterms:title "GeoSPARQL Ontology" ;
+	dcterms:publisher [
+		a sdo:Organization ;
+		sdo:name "Open Geospatial Consortium" ;
+		sdo:url "https://www.ogc.org"^^xsd:anyURI ;
+	] ;
+	dcterms:creator "OGC GeoSPARQL Standards Working Group" ;
+	dcterms:contributor [
+		a sdo:Person ;
+		sdo:name "Nicholas J. Car" ;
+		sdo:affiliation "SURROUND Australia Pty Ltd" ;
+		sdo:email "nicholas.car@surroundaustralia.com" ;
+		sdo:url <https://orcid.org/0000-0002-8742-7730> ;
+	] ,
+	[
+		a sdo:Person ;
+		sdo:name "Joseph Abhayaratna" ;		
+		sdo:affiliation "Geoscape Australia" ;
+		sdo:email "joseph.abhayaratna@geoscape.com.au" ;
+	] ;
+	dcterms:created "2020-09-09"^^xsd:date ;
+	dcterms:modified "2021-01-28"^^xsd:date ;
+	dcterms:replaces <http://www.opengis.net/ont/geosparql/1.0> ;
+	dcterms:description "An RDF/OWL vocabulary for representing spatial information"@en ;
+	dcterms:source <http://www.opengis.net/doc/IS/geosparql/1.1> , "OGC GeoSPARQL – A Geographic Query Language for RDF Data OGC 11-052r5"@en ;
+	rdfs:seeAlso <http://www.opengis.net/doc/IS/geosparql/1.1> ;
+	owl:versionInfo "OGC GeoSPARQL 1.1" .
+
 # #################################################################
 # #
 # #    Datatypes
 # #
 # #################################################################
-# 
-# 
-# http://www.opengis.net/ont/geosparql#gmlLiteral
 
-:gmlLiteral a rdfs:Datatype ;
-	dc:contributor "Matthew Perry" ;
-	dc:creator "OGC GeoSPARQL 1.0 Standard Working Group" ;
-	dc:date "2011-06-16"^^xsd:date ;
-	dc:description """A GML serialization of a geometry object."""@en ;
-	rdfs:comment """A GML serialization of a geometry object."""@en ;
-	rdfs:isDefinedBy <http://www.opengis.net/ont/geosparql> , <http://www.opengis.net/spec/geosparql/1.0> ;
-	rdfs:label "GML Literal"@en ;
+# http://www.opengis.net/ont/geosparql#gmlLiteral
+:gmlLiteral
```

Comment:
Is it appropriate to remove the existing micro-credits - Matt Perry, original date, etc?